### PR TITLE
display modal evolution tab content

### LINKED ISSUE #57: Modal - Evolution tab
**Is your feature request related to a problem? Please describe.**
Display pokemon evolution content in modal tab.

**Describe the solution you'd like**
To display pokemon evolution chain in the appropriate tab using pokemon sprites.

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
_List Changes Introduced by this PR_
1. Fetch all available evolution sprites from current pokemon modal.
2. Sort images to have evolution arrow in between pokemon sprites.
3. Display evolutions in order.

## Purpose
To find a way to search object tree from pokeapi to get pokemon evolutions, and then fetch to get sprites, then organize to show arrows.

## Learning
Used algorithm from stackoverflow to search for pokemon evolution names. 
https://stackoverflow.com/questions/39112862/pokeapi-angular-how-to-get-pokemons-evolution-chain

Closes #57

### CODE REVIEW

File: src/components/EvolutionTab.js [hunk 1]
```diff
@@ -111,7 +112,25 @@ const EvolutionTab = ({ evolutionChainUrl }) => {
...
+      organizedArray.push(spritesList[index]);
+      // if sprite is the last in list, then just return sprite without arrow
+      if (index !== spritesList.length - 1) {
+        organizedArray.push(evolutionImg());
+      }
+    }
+    return organizedArray;
+  }
+
+  return <div className="spriteContainer">{spritesWithEvolutionArrows()}</div>;
```
Comment (BlackCubes) [inline src/components/EvolutionTab.js:133]: Looking good!

File: src/components/PokeCard.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ const PokeCard = ({
       <Modal
         className="modalWindow"
         isOpen={modalIsOpen}
-        moveSet={moveSet}
+        // moveSet={moveSet}
```
Comment (BlackCubes) [inline src/components/PokeCard.js:79]: Do ye need it if it is commented out? If not, possibly remove?

**Review by BlackCubes — APPROVED** [review]
Looks good!

File: src/components/PokeCard.js (hunk 2 above)
Comment (Lap343) [inline src/components/PokeCard.js:79]: are you going to need this comment later on?

File: src/styles/components/_modal.scss [hunk 3]
```diff
@@ -106,6 +106,28 @@ $md-bg: #f9cf30;
...
+.evolution-sprites {
+  width: 175px;
+  height: 175px;
+  border: 1px dashed green;
+}
+
+// .evolutionImg {
+//   width: 45px;
+//   height: 45px;
+// }
```
Comment (Lap343) [inline src/styles/components/_modal.scss:129]: Same with these comments. Do they need to be there

**Review by Lap343 — APPROVED** [review]
Good code just some comments that need to be taken out unless you need them there then Saul Goodman

**Review by JorgeLVilla — APPROVED** [review]
Aside from the comments everything looks great. Nice job Danny!

File: src/components/EvolutionTab.js (hunk 1 above)
Comment (krferrell) [reply src/components/EvolutionTab.js:133]: Nice job here!

**Review by krferrell — APPROVED** [review]
Nicely done! Keep up the good work and don't forget to check for comments!

**Review by BlackCubes — COMMENTED** [review]
Nice changes!

**Pull request merged**